### ISSUE TITLE
holding Ctrl + hitting V repeatedly does not cycle through saved items

### ISSUE THREAD

**Issue by szepeviktor** [ffMathy/Shapeshifter#45]
In v6.0.0.1511 on Windows 10 Pro

**Comment by szepeviktor** [comment]
This is the nicest clipboard manager I've ever seen.
Mainly because one must only press <kbd>Ctrl + V</kbd> nothing more.

**Comment by ffMathy** [comment]
Icons are being handled in https://github.com/ffMathy/Shapeshifter/issues/22.

As for the "holding CTRL + Hitting V repeatedly does not cycle through saved items", can you clarify what you expect it to do in more detail?

I doubt the new Shapeshifter will have a tray icon. In general, tray icons are dying. However, if there is no other way, we will have one. Do you think it adds value?

The description will surely be fixed in the future.

As for the multi-instance problem, this will also be fixed soon.

Can I ask you to take all of these items on your list and create them as separate issues? That way, I can track them individually.

**Comment by szepeviktor** [comment]
> As for the "holding CTRL + Hitting V repeatedly does not cycle through saved items", can you clarify what you expect it to do in more detail?

1. As in 5.0: hold down Ctrl
1. Hit V (keep holding down Ctrl)
1. See the list of clipboard items
1. Hit V till you get your item
1. Release Ctrl

**Comment by szepeviktor** [comment]
@ffMathy Please reopen this renamed issue.

**Comment by szepeviktor** [comment]
Using up&down does not scroll the list.

**Comment by ffMathy** [comment]
Wait, Shapeshifter 5 cycles through items when tapping V while holding down CTRL? That's at least not on purpose.

But nice idea. However, it would conflict with being able to double paste quickly (hold down CTRL and then spam V repeatedly to paste many times).

**Comment by szepeviktor** [comment]
> Wait, Shapeshifter 5 cycles through items when tapping V while holding down CTRL? That's at least not on purpose.

That is why I use Shapeshifter.

One-handed solution.

**Comment by KoalaBear84** [comment]
I think a tray-icon is a good thing. You can then see that it is running and can get to the "Settings". Or you will have to find another good way to open the settings somewhere when holding down CTRL+V.

**Comment by ffMathy** [comment]
About the cycling. Since #33 is impeded at the moment, would this task be a nice substitute? I don't think they can both be features offered at the same time.

Although right now I like the multi paste one more.

**Comment by KoalaBear84** [comment]
I also like multi pasting the last one is more commenly used for me, and is also not affecting the normal clipboard behaviour.

**Comment by ffMathy** [comment]
Fair enough. Closing this as "won't fix". We can't satisfy everyone unfortunately.

**Comment by szepeviktor** [comment]
Could you add an option for one-handers? I mean using CTRL+V to cycle through clipboard items. #48

**Comment by ffMathy** [comment]
That's a great idea.

**Comment by thegabriestein** [comment]
@ffMathy, adding some way to support one handed it potentially really useful in many cases. But isn't releasing Control or V the primary method of initiating a paste command for the currently selected item? Potentially something to integrate with AHK as mentioned here in #223.

**Comment by ffMathy** [comment]
I still feel like this is a good idea.

**Comment by thegabriestein** [comment]
What about options to not autohide the overlay after Control + V release? And it's ignored either through:
1. Clicking / selecting an item with arrows 
2. Changing focus to another window, putting it in the background 
3. Clicking a close window button, or anywhere except the overlay
4. Even making it sticky in front of other windows, and it slides out from the edge of the screen when hovered?

Then you could use Control and/or V for other actions once the overlay is open, since they aren't what's keeping the overlay visible?
